jablib/src/main/java/org/jabref/model/entry/field/FieldTextMapper.java

```diff
@@ -0,0 +1,45 @@
+package org.jabref.model.entry.field;
+
+import java.util.StringJoiner;
+
+import org.jabref.model.strings.StringUtil;
+
+public class FieldTextMapper {
+    public static String getDisplayName(Field field) {
+        if (field.isStandardField()) {
+            StandardField standardField = (StandardField) field;
+            return switch (standardField) {
+                case DOI ->
+                        "DOI";
+                case ISBN ->
+                        "ISBN";
+                case ISRN ->
+                        "ISRN";
+                case ISSN ->
+                        "ISSN";
+                case PDF ->
+                        "PDF";
+                case PMID ->
+                        "PMID";
+                case PS ->
+                        "PS";
+                case URI ->
+                        "URI";
+                case URL ->
+                        "URL";
+                default ->
+                        StringUtil.capitalizeFirst(field.getName());
+            };
+        }
+
+        return field.getName();
+    }
+
+    public static String getDisplayName(OrFields fields) {
+        StringJoiner joiner = new StringJoiner("/");
+        for (Field field : fields.getFields()) {
+            joiner.add(StringUtil.capitalizeFirst(FieldTextMapper.getDisplayName(field)));
```

Comment:
We just need the display  name - no more other configuration.

```suggestion
            joiner.add(FieldTextMapper.getDisplayName(field));
```